modules/command/src/main/java/org/apache/fluo/command/FluoRemove.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.command;
+
+import org.apache.fluo.api.config.FluoConfiguration;
+import org.apache.fluo.core.client.FluoAdminImpl;
+
+public class FluoRemove {
+
+  public static void main(String[] args) {
+
+    CommonOpts opts = CommonOpts.parse("fluo remove", args);
+
+    FluoConfiguration config = CommandUtil.resolveFluoConfig();
+    config.setApplicationName(opts.getApplicationName());
+    opts.overrideFluoConfig(config);
+    config = FluoAdminImpl.mergeZookeeperConfig(config);
+
+    try (FluoAdminImpl admin = new FluoAdminImpl(config)) {
+      if (admin.applicationRunning()) {
```

Comment:
I think this code should be moved into  `FluoAdminImpl.remove()` method.  That way when someone calls `remove()` from the API, it also does this check to see if running.  However when moving to FluoAdminImpl, it should throw a FluoException instead of calling `System.exit(-1)`

Reply:
Agree! I'm fixing this.

Reply:
The FluoInit has this exactly check too. But I can't see a way to fix there without a refactoring.
Are there any chance we transform the command module in a facade without any logic inside? Moving this logic to  FluoAdminImpl.